language/stdlib/modules/libra.move

```diff
@@ -262,6 +262,40 @@ module Libra {
         };
     }
 
+    ///////////////////////////////////////////////////////////////////////////
+    // Treasury Compliant specific methods for DDs
+    ///////////////////////////////////////////////////////////////////////////
+
+    // Publish `preburn` under 'account'. Used for bootstrapping designated dealer
+    // as assocation TC account is creating this resource for DD
+    public fun publish_preburn_to_account<Token>(creator: &signer, account: &signer) {
+        assert_account_is_blessed(creator);
+        let preburn = Preburn<Token> { requests: Vector::empty(), is_approved: true };
+        move_to(account, preburn)
+    }
+
+
+    /// Send coin to the preburn holding area for `dd_address`, where it will wait to either be burned
+    /// or returned to the balance of `account`.
+    /// Fails if `dd_address` does not have a published DesignatedDealerPreburn resource
+    fun preburn_coin<CoinType>(tc_account: &signer, dd_address: address, coin: T<CoinType>)
```

Comment:
I'm confused about this --isn't it the DD's job to preburn and the TC's job to approve the burn or cancel?

Reply:
yes, i was wrong. Changed now